queries/goctl/highlights.scm

```diff
@@ -0,0 +1,72 @@
+; Key Symbol
+[
+  "."
+  ","
+  ":"
+  ";"
+] @punctuation.delimiter
+
+[
+  "("
+  ")"
+  "{"
+  "}"
+  "["
+  "]"
+] @punctuation.bracket
+
+; Keywords
+[
+  "syntax"
+  "info"
+  "service"
+] @keyword
+
+"import" @keyword.import
+
+"returns" @keyword.return
+
+"type" @keyword.type
+
+[
+  "@doc"
+  "@handler"
+  "@server"
+] @keyword.directive
+
+; Service
+(serviceName) @type
+
+; Httpmethod
+(HTTPMETHOD) @operator
```

Comment:
I think `@keyword.operator` is more fitting here

Reply:
This should be fixed.